ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -330,7 +318,7 @@ short getVersion() {
       if (!mapped.get()) {
         ConcurrentSkipListSet<IndexValue> values = index.get(keyToFind);
         if (values != null) {
-          metrics.blobFoundInActiveSegmentCount.inc();
+          metrics.blobFoundInMemSegmentCount.inc();
```

Comment:
Is this metric any useful if the entire index is in memory? Wouldn't you rather report whether blob was found in the active (non-sealed) segment or not?

Reply:
This metric is meant to be useful only when some segments are mmapped. An `IndexSegment` does not seem to know whether it is the "active" segment (something that is not active may not be sealed yet). So I don't know if it makes sense to separate it by sealed v/s non sealed. It is true that is no use at all if the index is fully in memory.